embedded/ahtree/ahtree.go

```diff
@@ -779,25 +779,19 @@ func (t *AHtree) Close() error {
 
 	t.closed = true
 
-	errors := make([]error, 0)
+	merrors := multierr.NewMultiErr()
 
-	pErr := t.pLog.Close()
-	if pErr != nil {
-		errors = append(errors, pErr)
-	}
+	err := t.pLog.Close()
+	merrors.Append(err)
```

Comment:
Is it really OK to continue if there was an error with the close?

If there's an error during flush of payload or digests then maybe it would be better to discard flushing commit log too?

Reply:
consistency does not relies on Close. The intention is to properly close as many files as possible.

Not in the scope on this PR, but if we find a different way on handling close operation, it'll be ok of course